backend/configuration_controller/config.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
I think that we can remove `backend` directory

Reply:
removed `backend` and moved its content one level up